requirements.txt

```diff
@@ -5,7 +5,7 @@ loguru
 numpy
 Pillow
 pyperclip
-torch>=1.0
+torch>=2.0.0
```

Comment:
The version installed by pip is defined in setup.py, so this change isn't sufficient. But anyway I think that maybe it'd be better to just add a notice in readme, that pytorch>=2.0 is required for MPS and leave the requirements as they are. I don't want to force the pytorch upgrade on users who don't need it.

Reply:
Okay, just adding a notice in readme will work just fine.